examples/with-tuist/tuist-workspace/README.md

```diff
@@ -0,0 +1,44 @@
+# Tuist iOS Example
+
+This is an example project demonstrating how to use the `cargo-xcframework` plugin with a Tuist generated iOS project.
+
+## Getting Started
+
+1. **Install Tuist**: If you haven't already, first [install Tuist](https://docs.tuist.io/guide/introduction/installation.html#installation).
+2. **Build and Run**: Use the `tuist run TuistIos` command, just like `cargo run`. Cool, right?
```

Comment:
When running this from the `tuist-workspace` dir I get:
```
❯ tuist run TuistIos
Generating project for running
Loading and constructing the graph
It might take a while if the cache is empty
Generating workspace CargoXCFramework.xcworkspace
Generating project Project
Couldn't find scheme TuistIos. The available schemes are: App.
Consider creating an issue using the following link: https://github.com/tuist/tuist/issues/new/choose
```

If I run it with `tuist run App` I get:
```
❯ tuist run App
Loading and constructing the graph
It might take a while if the cache is empty
Only single platform targets supported. The target App specifies multiple supported platforms (macos, ios).
Consider creating an issue using the following link: https://github.com/tuist/tuist/issues/new/choose
```

Reply:
Ops, sorry for the confusion caused by the outdated readme. I've migrated `tuist-ios` to `tuist-workspace` to support multiple platforms. I will update it and address the suggestion comments as well.